test/test_frase.py

```diff
@@ -206,7 +206,7 @@ def test_gen(self):
     # custom_header_support
     def test_gen_20180801(self):
         fg=FraseGen()
-        ft = fg.gen('https://space.ktmrmshk.com/','space.ktmrmshk.com.edgekey.net', 'space.ktmrmshk.com.edgekey-staging.net',  {'User-Agent': 'iPhone', 'Debug-abc':'foobar' })
+        ft = fg.gen('https://space.ktmrmshk.com/','space.ktmrmshk.com.edgekey.net', 'space.ktmrmshk.com.edgekey-staging.net',  {'User-Agent': 'iPhone', 'Debug-abc':'foobar' }, {'strict_redirect_cacheability': False, 'strict_check_cacheability': True})
```

Comment:
due to `def gen(self, url, src_ghost, dst_ghost, headers={}, option={}, mode={}):`, 
it must be 
```
ft = fg.gen('https://space.ktmrmshk.com/','space.ktmrmshk.com.edgekey.net', 'space.ktmrmshk.com.edgekey-staging.net',  {'User-Agent': 'iPhone', 'Debug-abc':'foobar' }, mode={'strict_redirect_cacheability': False, 'strict_check_cacheability': True})
```